content/rdi/write-behind/configuration-guide.md

```diff
@@ -0,0 +1,99 @@
+        # query_args:       # optional parameters for SQL query execution - typically not required for RDI operation
+```
+
+**For non-relational datastores:**
+```yaml
+connections:
+    my-nosql-datastore:
+        type: <db-type>     # cassandra
+        hosts: <db-hosts>   # array of IP addresses or host names of a datastore nodes
+        port: <db-port>     # database port
+        database: <db-name> # name of the database
+        user: <db-user>         # database user
+        password: <db-password> # database password
+```
+
+## Microsoft SQL Server
+
+**Microsoft SQL Server** supports different authentication mechanisms (SQL Server Authentication, Integrated Windows Authentication) and protocols (NTLM,  Kerberos). RDI can use all of them, however some additional configuration might be required for systems that should utilize Kerberos.
+
+### Account Permissions
+
+To enable RDI to work with a SQL Server database, you need to ensure that the account you specify has been assigned the `db_datawriter` role at a minimum.
+
+### SQL Server Authentication
+
+To use SQL Authentication mode, you need to create a SQL user with login credentials, and then assign the necessary permissions to that user for the target database as described earlier.
+
+```yaml
+connections:
+    mssql2019-sqlauth:
+        type: sqlserver
+        host: ip-10-0-0-5.internal
+        port: 1433
+        database: rdi_wb_database
+        user: rdi_user
+        password: secret
+```
+
+### Windows Authentication
+
+To use Windows Authentication mode, you need to create a Windows or AD account that has the necessary permissions to access the target database and is able to log into SQL Server. The Linux machine hosting RDI can be configured to support NTLM or Kerberos authentication protocols. Please consult with your security team as the configuration settings will vary.
+
+**For NTLM:**
+```yaml
+connections:
+    mssql2019-ntlm:
+        type: sqlserver
+        host: ip-10-0-0-5.internal
+        port: 1433
+        database: rdi_wb_database
+        user: MYDOMAIN\rdi_service_account  # company-domain\service-account
+        password: secret                    # NTLM requires to provide a password
+```
+
+> Note: User must be specified with the domain name for Windows Authentication to work
+
+**For Kerberos:**
+
+* Please contact Redis team
+
+After configuring the RDI connection and deploying the write-behind job, the operator can check whether RDI is using the expected authentication mechanism and protocol by running the following SQL query (note, this may require `sysadmin` role):
+
+```sql
+SELECT session_id, auth_scheme FROM sys.dm_exec_connections;
+```
+
+The results will indicate which `auth_scheme` is used by each session and may take the following values: `SQL`, `NTLM` and `Kerberos`.
+
+### Azure Active Directory
+
+* Please contact Redis team
```

Comment:
@redima consider adding contact info

Reply:
same as above. let's see what Yaron will reply with.

Reply:
@redima please remove the kerberos & Azure AD section (or say not supported for now)